pages/common/find.md

```diff
@@ -25,3 +25,11 @@
 - delete files by name, older than a certain number of days
 
 `find {{root_path}} -name {{'*.py'}} -mtime {{-180d}} -delete`
+
+- find empty files
+
+`find -empty`
+
+- find files matching more than one search criteria
+
+`find {{root_path}} -name {{'*.py'}} -or -name {{'*.rb'}}
```

Comment:
I'm glad to contribute, I've also learned a few importand command options thanks to this project. :)